### ISSUE TITLE
Remove hover state on disabled fields in CMS

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#12174]
## Description
On disabled fields in the CMS, there is a hover state which makes it seem that this field might be actionable. As this is not true, this state is confusing and misleading.

### Additional context
I have seen 2 types of disabled fields so far, input fields and checkboxes. The Input fields so not have a hover state, but the checkboxes do - they turn blue on hover. There could possibly be other types of fields within the CMS that I just haven't seen yet
![image](https://user-images.githubusercontent.com/106678594/210654642-263ced3f-bced-46d4-bf6b-f79ad820bb3f.png)
Both of these fields can be seen on the VAMC Facility edit page.

## Acceptance Criteria
- [ ] Remove hover state from all disabled fields throughout the CMS
- [ ] UX review
- [ ] Accessibility review

### CMS Team
Please check the team(s) that will do this work.

- [ ] `Program`
- [ ] `Platform CMS Team`
- [ ] `Sitewide Crew`
- [x] `⭐️ Sitewide CMS`
- [ ] `⭐️ Public Websites`
- [ ] `⭐️ Facilities`
- [ ] `⭐️ User support`

**Comment by BlakeOrgan** [comment]
Yeah, I agree. Disabled fields should not have hover states. Good catch!
